Documentation/QuickInstall/Index.rst

```diff
@@ -17,13 +17,19 @@ machine (which needs elevated permissions on Windows machines).
 
 1. Checkout out the system requirements
 
-   Install required software, such as:
+   Install required server software, such as:
 
    * Web server (Apache, Nginx, IIS...)
    * PHP ≥ 7.2
    * Database server (MySQL, PostgreSQL, MariaDB ...)
+   * Image Magick or Graphics Magick   
+   * Open SSL. Under Windows you must add manually the system environment variable, e.g.:
+          *OPENSSL_CONF = C:\Web\Xampp\Xamp-7.3.28\apache\conf\openssl.cnf*
```

Comment:
Do we have a page where we can link to for this? I'd rather not have something as specific as this in our general docs. Especially not with fixed versions.

Reply:
I have found this wiki:  https://wiki.openssl.org/index.php/Command_Line_Utilities 

And here again is a documentation about openssl under Window, and it is missing the important example. Only Windows experts will know what to do.

Reply:
You could additionally link here:
https://adamtheautomator.com/openssl-windows-10/

However it would be enough to just only do the one step I have added here. This would save the time for reading of many chapters.

Reply:
"e.g.:" => "for example:"